app/geo/gui/src/ts/twxplore/gui/geo/reducers/map/mapReducer.ts

```diff
@@ -62,6 +61,15 @@ export const mapReducer = (state: MapState, action: BaseAction): MapState => {
         result.featuresByType[
           feature.type! as keyof typeof result.featuresByType
```

Comment:
I see the same expression (result.featuresByType[...]) over and over again, please put into a variable.